src/silo/query_engine/actions/aggregated.cpp

```diff
@@ -25,16 +25,19 @@
 using silo::query_engine::CopyOnWriteBitmap;
 
 namespace {
+using silo::query_engine::actions::GroupByField;
 
-std::vector<silo::schema::ColumnIdentifier> parseGroupByFields(
+const std::string GROUP_BY_FIELDS_FIELD_NAME = "groupByFields";
+
+std::vector<silo::schema::ColumnIdentifier> bindGroupByFields(
    const silo::schema::TableSchema& schema,
-   const std::vector<std::string>& group_by_fields
+   const std::vector<GroupByField>& group_by_fields
 ) {
    std::vector<silo::schema::ColumnIdentifier> group_by_metadata;
-   for (const std::string& group_by_field : group_by_fields) {
-      auto column = schema.getColumn(group_by_field);
+   for (const GroupByField& group_by_field : group_by_fields) {
+      auto column = schema.getColumn(group_by_field.name);
       CHECK_SILO_QUERY(
-         column.has_value(), "Metadata field '{}' to group by not found", group_by_field
+         column.has_value(), "Metadata field '{}' to group by not found", group_by_field.name
```

Comment:
(Possibly in another PR): We could actually tell which fields would be available (currently it's quite impossible in SILO to find that out without looking into the config).